dts/arm/infineon/psoc6/mpns/CY8C6336BZI_BLD14.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include "../psoc6_01/psoc6_01.124-bga-sip.dtsi"
+
+
+/ {
+
+
+
```

Comment:
can you cleanup the whitespace here (remove blank lines), and in other similar files.

Reply:
updated..